plugin/rxnorm-transformation-maven-plugin/src/main/java/dev/ikm/maven/RxnormTransformationMojo.java

```diff
@@ -81,8 +73,8 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         this.namespace = UUID.fromString(namespaceString);
         File datastore = new File(datastorePath);
-        this.rxnormModule = RxnormUtility.makeConceptProxy(namespace, rxnormModuleStr);
-        this.rxnormAuthor = RxnormUtility.makeConceptProxy(namespace, rxnormAuthorStr);
+        this.rxnormModule = EntityProxy.Concept.make(PublicIds.of(UUID.fromString("ae4818f8-d523-48e8-abf9-099237ae01ab")));
```

Comment:
I'm assuming these hardcoded UUIDs represent concepts from the authored starter data, and that's why we don't use the uuid generation with the namespace. If so, can we make static class field vars that capture the meaning of each of these?